src/test/java/org/nibor/autolink/AutolinkWwwUrlTest.java

```diff
@@ -0,0 +1,60 @@
+package org.nibor.autolink;
+
+import java.util.Arrays;
+import java.util.EnumSet;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameter;
+import org.junit.runners.Parameterized.Parameters;
+
+@RunWith(Parameterized.class)
+public class AutolinkWwwUrlTest extends AutolinkUrlTest {
```

Comment:
When we have a new `LinkType` for www, you won't be able to inherit from `AutolinkUrlTest` anymore.